README.md

```diff
@@ -7,144 +7,119 @@
-- developer-docs
-
-Open a shell within developer-docs and run:
-```shell
-make
-```
+If the content you're changing is in the OpenAPI spec, you should make the
+changes within codez. However, if you want to quickly test something, you can
+make the changes in def/asana_oas.yaml. Just remember to put the changes in
+codez if you want them to not be overridden. Then, you should run the `make`
+command above. I prefer to run `git diff` after doing so to see the generated
+changes. Run `bundle exec middleman server` if it's not already running, and go
+to the url it provides to check out the changes you made.
 
-To see what this command does, read the Makefile in this repo.
+Merging these changes into master causes them to be deployed.
 
-Then to generate the html from the markdown, we use [slate](https://github.com/lord/slate). This happens automatically when the middleman starts the server.
-```shell
-# either run this to run locally
-bundle exec middleman server
+## Making Changes to Styles
 
-# OR run this to run with vagrant
-vagrant up
-```
+Make changes in source/stylesheets/*\_variables.scss* because the changes here
+will be valid with future versions of Slate.
 
-*Why did we fork widdershins?* For our use case, we needed things like denormalizing and dereferencing. We tried doing this to the spec & using an unforked widdershins, but as we progressed with client library generation, it made more sense to keep a clean spec and do this doc-specific editing in the tooling. A potential future is pulling out this logic to a "openapi spec transformer" to prep the spec for widdershins, but there will be a trade-offs to consider.
+If you need to make more complex css changes, edit **screen.css.scss** or
+**print.css.scss** but keep in mind that these will need to be merged for new
+versions of Slate.
 
-## Making content changes
-_Internal Asanas: See https://app.asana.com/0/77076599077/1122503737028047/f and https://app.asana.com/0/0/1200652548580470/f before making any updates._
+## Deployment (GitHub Pages)
 
-If the content you're changing is static (not generated from the OpenAPI spec), you'll edit the `.md` files in source/includes/markdown.
+This should happen automatically when changes are merged into this repo.
 
-If the content you're changing is in the OpenAPI spec, you should make the changes within codez. However, if you want to quickly test something, you can make the changes in def/asana_oas.yaml. Just remember to put the changes in codez if you want them to not be overridden.
-Then, you should run the `make` command above.
-I prefer to run `git diff` after doing so to see the generated changes.
-Run `bundle exec middleman server` if it's not already running, and go to the url it provides to check out the changes you made.
+If you need to do this manually, then run the `deploy.sh` script. This will use
+your local git credentials and local /build folder to push a build to a branch
+named gh-pages (Where the docs are hosted).
```

Comment:
On the topic of bolding paths, **/build** could benefit from a quick bold here too!